interfaces/bots/interface_bot.py

```diff
@@ -167,26 +164,16 @@ def get_command_portfolio():
 
         portfolios_string = ""
         if has_real_trader:
-            portfolios_string += "{0}Portfolio value : {1} {2}{3}".format(
-                REAL_TRADER_STR,
-                PrettyPrinter.get_min_string_from_number(portfolio_real_current_value),
-                reference_market,
-                EOL)
-            portfolios_string += "{0}Portfolio : {2}{1}{2}{2}".format(
-                REAL_TRADER_STR,
-                PrettyPrinter.global_portfolio_pretty_print(real_global_portfolio),
-                EOL)
+            portfolios_string += f"{REAL_TRADER_STR}Portfolio value : " \
```

Comment:
I think that a [_Multi-line string syntax_](https://www.smallsurething.com/multi-line-strings-in-python/) will be better here. (with `"""` )